kustomize/install/bases/rbac/cluster/role_binding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
   name: postgres-operator
 subjects:
 - kind: ServiceAccount
-  name: postgres-operator
+  name: pgo
```

Comment:
🔧 There should be a similar change in `kustomize/install/bases/rbac/namespace/role_binding.yaml`.

See https://access.crunchydata.com/documentation/postgres-operator/v5/installation/kustomize/#installation-mode.

Reply:
:+1: updated, thanks.